tests/TestCase/Cache/Engine/MemcachedEngineTest.php

```diff
@@ -446,6 +446,18 @@ public function testConnectIpv6()
     }
 
     /**
+     * test domain starts with u
+     *
+     * @return void
+     */
+    public function testParseServerStringWithU()
+    {
+        $Memcached = new TestMemcachedEngine();
+        $result = $Memcached->parseServerString('udomain.net:13211');
+        $this->assertEquals(array('udomain.net', '13211'), $result);
```

Comment:
A sniff for it got added to [phpcs](https://github.com/squizlabs/PHP_CodeSniffer/commit/081f49aa967e2cd7c579442ffa05ef0193c6018a) just hours ago, so just need to wait until it's next release :smile: